app/graphql/mutations/create_asset.rb

```diff
@@ -36,12 +36,12 @@ def initialize_child_asset(parent, file_name)
     asset
   end
 
-  def initialize_asset_resource_for_blob(asset, blob, storage)
```

Comment:
I don't understand the reason for this change (it looks like the parameterized reference has been exchanged for a static reference in the method) and it seems like it will make it somewhat more difficult to test. Is there a context for it?

Reply:
It seemed like an opportunity to remove an unneeded param and match with the way that`Hyacinth::Config.resource_storage` was called in other places in the app (and the associated test modification wasn't complicated), but if you'd prefer the previous way I'm glad to change it back.